wit/consumer.wit

```diff
@@ -1,17 +1,19 @@
+/// The consumer interface allows a guest to dynamically update its subscriptions and configuration
+/// as well as functionality for completing (acking) or abandoning (nacking) messages.
 interface consumer {
-    // {client, message, channel, error, guest-configuration}
-    use messaging-types.{client, message, channel, error, guest-configuration};
-
-    /// Blocking receive for t-milliseconds with ephemeral subscription – if no message is received, returns None
-    subscribe-try-receive: func(c: client, ch: channel, t-milliseconds: u32) -> result<option<list<message>>, error>;
```

Comment:
I removed these and replaced them in the new `request-reply` interface as components should handle messages via the `handle` function in `guest` and these are more likely to be used for request/reply operations. If we do want these, let's capture the use cases and probably put them in a separate interface